conda_forge_tick/migrators/nvtools.py

```diff
@@ -67,48 +67,15 @@ def _insert_subsection(
 
 
 class AddNVIDIATools(Migrator):
-    """Add the cf-nvidia-tools package to NVIDIA redist feedstocks.
-
-    In order to ensure that NVIDIA's redistributed binaries (redists) are being packaged
-    correctly, NVIDIA has created a package containing a collection of tools to perform
-    common actions for NVIDIA recipes.
-
-    At this time, the package may be used to check Linux binaries for their minimum glibc
-    requirement in order to ensure that the correct metadata is being used in the conda
-    package.
-
-    This migrator will attempt to add this glibc check to all feedstocks which download any
-    artifacts from https://developer.download.nvidia.com. The check involves adding
-    "cf-nvidia-tools" to the top-level build requirements and something like:
-
-    ```bash
-    check-glibc "$PREFIX"/lib/*.so.* "$PREFIX"/bin/*
-    ```
-
-    to the build script after the package artifacts have been installed.
-
-    > [!NOTE]
-    > A human needs to verify that the glob expression is checking all of the correct
-    > artifacts!
-
-    > [!NOTE]
-    > If the recipe does not have a top-level requirements.build section, it should be
-    > refactored so that the top-level package does not share a name with one of the
-    > outputs. i.e. The top-level package name should be something like "libcufoo-split".
-
-    More information about cf-nvidia-tools is available in the feedstock's
-    [README](https://github.com/conda-forge/cf-nvidia-tools-feedstock/tree/main/recipe).
-
-    Please ping carterbox for questions.
-    """
+    """Add the cf-nvidia-tools package to NVIDIA redist feedstocks."""
 
     name = "NVIDIA Tools Migrator"
 
     rerender = True
 
     max_solver_attempts = 3
 
-    migrator_version = 0
+    migrator_version = 1
```

Comment:
This will reissue all PRs the migrator ever made. Did you mean to do that?

We almost never due that because it is a bit spammy.

Reply:
Think that is ok. We are the ones creating and receiving the spam

Reply:
This is good to know for the future. Luckily only a few PRs had been opened, so there were few duplicates. How do you update an in-progress migrator, then? i.e. have the bot rerun unedited pull requests?

Reply:
Well you have two choices. 

1. Leave the migrator_version as is and the bot will use the updates for new PRs, but leave old ones alone. 
2. Bump the migrator_version and get new PRs for everything.

There is no feature to go back and edit pull requests already issued.